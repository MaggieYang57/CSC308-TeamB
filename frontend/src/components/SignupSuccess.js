import React from "react";
import { Link } from "react-router-dom";
import "bootstrap/dist/css/bootstrap.min.css";
import "../css/Signup.css";

function SignupSuccess(props) {
  return (
    <div className="signup-form">
      <div id="title-signup">
        <b
          className="text text-center "
          style={{ fontSize: 40, color: "#2C6674" }}
        >
          Successfully Signed Up!
        </b>
      </div>
      <p style={{ marginTop: 20 }}>
        Please <Link to="/login">log in </Link>to continue
      </p>
    </div>
  );
}

export default SignupSuccess;
