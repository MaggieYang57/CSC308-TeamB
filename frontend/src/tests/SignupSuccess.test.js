import React from 'react';
import { MemoryRouter } from 'react-router';
import renderer from 'react-test-renderer';
import SignupSuccess from '../components/SignupSuccess';

it('renders correctly', () => {
  const tree = renderer
    .create(<MemoryRouter> <SignupSuccess /> </MemoryRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});