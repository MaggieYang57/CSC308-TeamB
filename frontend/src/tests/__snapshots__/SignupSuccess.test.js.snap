// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  " ",
  <div
    className="signup-form"
  >
    <div
      id="title-signup"
    >
      <b
        className="text text-center "
        style={
          Object {
            "color": "#2C6674",
            "fontSize": 40,
          }
        }
      >
        Successfully Signed Up!
      </b>
    </div>
    <p
      style={
        Object {
          "marginTop": 20,
        }
      }
    >
      Please 
      <a
        href="/login"
        onClick={[Function]}
      >
        log in 
      </a>
      to continue
    </p>
  </div>,
  " ",
]
`;
